community/modules/internal/slurm-gcp/internal_instance_template/main.tf

```diff
@@ -120,7 +120,7 @@ resource "google_compute_instance_template" "tpl" {
     nic_type           = local.nic_type
     stack_type         = var.stack_type
     dynamic "access_config" {
-      for_each = var.access_config
+      for_each = coalesce(var.access_config, [])
```

Comment:
This seems like a little bit of a hack since default is already `[]`. 

I suspect this is because someone upstream is setting variable to null. I believe the correct thing to do here is to set `nullable = false` on the input variable and fix the upstream bad actor. 

Would you be willing to attempt this. If it proves problematic then we could move forward as you propose.